content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2579,10 +2581,11 @@ In [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and later, Pri
 
 In the following example, `Tag` is ignored because it does not have a valid unique identifier:
 
-```prisma highlight=5,10;normal
+```prisma highlight=5;normal
```

Comment:
Ugh that was pretty wrong before

Reply:
@janpio yeah, another point in favor of the inline highlight annotations